### PR TITLE
feat: add router

### PR DESCRIPTION
**Pull request by mwdchang**
### Summary
- Adding router setup
- Remove template/boiler-plates from starter kit

## Test
- yarn 
- yarn dev
- http://localhost:5173/ should work with a placeholder message

### CODE REVIEW

File: packages/client/index.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
```
Comment (YohannParis) [inline packages/client/index.html:5]: We can remove the icon as well.
Don't we have an Uncharted one while waiting to make a TERArium icon?

File: packages/client/src/views/Home.vue [hunk 2]
```diff
@@ -0,0 +1,7 @@
+i<script>
```
Comment (YohannParis) [inline packages/client/src/views/Home.vue:1]: random `i`.
you can remove the whole `<script>` tag as well.

**Review by YohannParis — APPROVED** [review]

File: packages/client/src/views/Home.vue (hunk 2 above)
Comment (mwdchang) [reply packages/client/src/views/Home.vue:1]: Random `i` is a vim artifact, sometimes you will also see random Js and random Ks 😆

**Review by jryu01 — CHANGES_REQUESTED** [review]

**Comment by mwdchang** [comment]
@jryu01 I have linter warnings but I do not have transpile/compile errors.

`vue/multi-word-component-names` is a little bit annoying.

**Review by jryu01 — APPROVED** [review]

**Pull request merged**